jvm/core/src/main/kotlin/com/intuit/playerui/core/flow/state/NavigationFlowState.kt

```diff
@@ -52,6 +53,25 @@ public class NavigationFlowActionState internal constructor(override val node: N
     internal object Serializer : NodeWrapperSerializer<NavigationFlowActionState>(::NavigationFlowActionState, ACTION.name)
 }
 
+/** Similar to NavigationFlowActionState but with an async enabled context */
+@Serializable(with = NavigationFlowAsyncActionState.Serializer::class)
+public class NavigationFlowAsyncActionState internal constructor(override val node: Node) :
+    NavigationFlowTransitionableState(node),
+    NodeWrapper {
+
+    override val stateType: NavigationFlowStateType = ASYNC_ACTION
+
+    /**
+     * An expression to execute.
+     * The return value determines the transition to take
+     */
+    public val exp: Expression by NodeSerializableField(Expression.serializer())
+
+    public val await: Boolean by NodeSerializableField(Boolean.serializer())
```

Comment:
does exposing this provide any value?

Reply:
Beyond having consistent multiplatform definitions, I could see it being useful during debugging, but probably not actually used for runtime logic